SwiftMonkey/SwiftMonkey.xcodeproj/project.pbxproj

```diff
@@ -183,7 +197,8 @@
 		OBJ_20 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				ENABLE_TESTABILITY = YES;
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
+				ENABLE_BITCODE = NO;
```

Comment:
Which is why I am worried about this pull request disabling it.

@JuGoo, can you explain why this change is here?